backend/commons/src/test/java/org/sonarsource/sonarlint/core/commons/GitBlameUtilsTest.java

```diff
@@ -21,49 +21,62 @@
 
 import java.io.IOException;
 import java.nio.file.Path;
+import java.util.List;
 import java.util.Set;
 import org.eclipse.jgit.api.Git;
 import org.eclipse.jgit.api.errors.GitAPIException;
 import org.eclipse.jgit.util.FileUtils;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
 import org.junit.jupiter.api.io.TempDir;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.jgit.util.FileUtils.RECURSIVE;
+import static org.junit.jupiter.api.condition.OS.WINDOWS;
 import static org.sonarsource.sonarlint.core.commons.testutils.GitUtils.commit;
 import static org.sonarsource.sonarlint.core.commons.testutils.GitUtils.createFile;
 import static org.sonarsource.sonarlint.core.commons.testutils.GitUtils.createRepository;
 
 class GitBlameUtilsTest {
 
-  @TempDir
-  private static Path baseDir;
   private Git git;
-  private Path gitDirPath;
+  @TempDir
+  private Path projectDir;
 
   @BeforeEach
   public void prepare() throws IOException {
-    gitDirPath = baseDir.resolve("gitDir");
-    git = createRepository(gitDirPath);
+    git = createRepository(projectDir);
   }
 
   @AfterEach
   public void cleanup() throws IOException {
-    FileUtils.delete(gitDirPath.toFile(), RECURSIVE);
+    FileUtils.delete(projectDir.toFile(), RECURSIVE);
   }
 
   @Test
   void it_should_blame_file() throws IOException, GitAPIException {
-    createFile(gitDirPath, "fileA", "line1", "line2", "line3");
+    createFile(projectDir, "fileA", "line1", "line2", "line3");
     var c1 = commit(git, "fileA");
 
-    var sonarLintBlameResult = GitBlameUtils.blameWithFilesGitCommand(git.getRepository(), Set.of(Path.of("fileA")));
+    var sonarLintBlameResult = GitBlameUtils.blameWithFilesGitCommand(projectDir, Set.of(Path.of("fileA")));
     var blameResult = sonarLintBlameResult.getBlameResult();
     assertThat(blameResult.getFileBlameByPath()).hasSize(1);
     assertThat(blameResult.getFileBlameByPath().get("fileA").getCommitDates())
       .hasSize(3)
       .allMatch(date -> date.equals(c1));
   }
+
+  @Test
+  @EnabledOnOs(WINDOWS)
```

Comment:
I am not entirely sure, but if we create a new folder under the project directory, the path should be in Windows format, right? If that's the case, we don't need to give the path in Windows format hardcoded. We could define the path as 
```java
projectDir.resolve("innerDir").resolve("fileA")
```

In that case, we don't need to use `@EnabledOnOs(WINDOWS)` and test it also in other OSs

Wdyt?